iocore/eventsystem/I_SocketManager.h

```diff
@@ -37,6 +37,14 @@
 #include "I_EventSystem.h"
 #include "I_Thread.h"
 
+#ifndef SOCK_NONBLOCK
+#define SOCK_NONBLOCK O_NONBLOCK
+#endif
+
+#ifndef SOCK_CLOEXEC
+#define SOCK_CLOEXEC O_CLOEXEC
```

Comment:
Pretty sure OS X doesn't have ``O_CLOEXEC``. Also the ``eventfd`` code uses ``FD_CLOEXEC`` where ``O_CLOEXEC`` would be more correct.